### ISSUE TITLE
List all potential test benchmarks

### ISSUE THREAD

**Issue by faneshion** [gomate-community/rageval#63]
List all most used datasets in RAG researches, and we will add them to the benchmarks.

- [ ] THUDM/webglm-qa from huggingface: load_dataset("THUDM/webglm-qa")

**Comment by FBzzh** [comment]
- [ ] MedMCQA  from hugginggace: https://huggingface.co/datasets/medmcqa
- [ ] GSM8K from hugginggace: https://huggingface.co/datasets/gsm8k
- [ ] BBH from github: https://github.com/suzgunmirac/BIG-Bench-Hard
- [ ] SQuAD from hugginggace: https://huggingface.co/datasets/squad
- [ ] SQuAD_v2 from hugginggace: https://huggingface.co/datasets/squad_v2
- [ ] Wizard-of-Wikipedia(WoW) from hugginggace: https://huggingface.co/datasets/chujiezheng/wizard_of_wikipedia

**Comment by Wenshansilvia** [comment]
Select and implement typical benchmarks, collect RAG papers that utilized these benchmarks, and try to reproduce evaluation result in the paper. 
1. List benchmark and related papers & metrics.
2. Produce testset using baseline RAG in the paper. Pack testset as dataset format and upload to HuggingFace.
3. Reproduce evaluation result in RAGEval. 

Eli5 @QianHaosheng , ASQA @bugtig6351 , Fever @henan991201
